opensrp-immunization/src/main/java/org/smartregister/immunization/fragment/VaccinationDialogFragment.java

```diff
@@ -460,7 +467,7 @@ public void onClick(View view) {
             }
         });
 
-        updateDateRanges(vaccinateToday, vaccinateEarlier, set, earlierDatePicker);
+        //updateDateRanges(vaccinateToday, vaccinateEarlier, set, earlierDatePicker);
```

Comment:
No need to update data ranges anymore?

Reply:
Nice catch! Must have been disabled when debugging, pushing the patch shortly